'main.css.scss_swapped'

```diff
@@ -0,0 +1,44 @@
+.navbar {
+  background-color: #4DB4D6;
+  width: 100%;
+  overflow: hidden;
+
+  ul {
+    list-style: none;
+
+    li {
+      display: inline;
+      padding: 0.8em;
+
+      a {
+        text-decoration: none;
+        color: white;
+      }
+    }
+  }
+}
+
+.links {
+  float: left;
+ }
+
+.auth {
+  float: right;
+}
+
+#wrapper {
+  background-color: #d9d9d9;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.details a {
+  //padding: 0.3em;
+}
+
+#links-list li {
+  padding: 0.5em;
+  a {
+    color: #4DB4D6;
+  }
+}
```

Comment:
woops I think that was a vim recovery file. I'm gonna delete it and probably also stop using vim...